lib/server.go

```diff
@@ -667,6 +673,20 @@ func (srv *Server) GetSnapshot(pp *Peer) {
 		"with Prefix (%v) and SnapshotStartEntry (%v)", pp, prefix, lastReceivedKey)
 }
 
+// FIXME: This is a temporary hack that we have to employ until we are confident nodes have
+//
+//	downloaded the latest code that sends an empty db chunk for a non-existent prefix. We
+//	check if the prefix is the newly-added PrefixGroupMembershipIndex and if so, filter it out.
+func (srv *Server) CheckIfStatePrefixExistsForBlockHeight(blockHeight uint64, prefix []byte) bool {
+	switch prefix[0] {
+	case Prefixes.PrefixGroupMembershipIndex[0]:
+		if uint32(blockHeight) < srv.blockchain.params.ForkHeights.DeSoAccessGroupsBlockHeight {
+			return false
+		}
+	}
+	return true
```

Comment:
```suggestion
	return prefix[0] != Prefixes.PrefixGroupMembershipIndex[0] || uint32(blockHeight) >= srv.blockchain.params.ForkHeights.DeSoAccessGroupsBlockHeight 
```
feels like we don't really need a switch-case statement here

Reply:
fixed